chcemvediet/apps/anonymization/cron.py

```diff
@@ -25,6 +33,47 @@ def attachment_normalization():
             content_type=content_types.PDF_CONTENT_TYPE
         )
         cron_logger.info(u'Normalized attachment: {}'.format(attachment))
+
+    elif attachment.content_type in content_types.LIBREOFFICE_CONTENT_TYPES:
+        directory_name = tempfile.mkdtemp(dir=u'.')
```

Comment:
`dir=u'.'` nie je dobry napad. Ktovie co je aktualny adresar v produkcii a ci tam vobec mame pravo zapisovat. Pravdepodobne to bude root naklonovaneho projektu. Urcite nie je dobre zapisovat temp subory do adresara, kde mame skripty z projektu. Idealne pouzi standadne systemove miesto, kde sa temp subory vytvaraju.

Reply:
:ok: